ansible/roles/fyrevm_provision/tasks/fyrevm_provision.yml

```diff
@@ -26,15 +26,17 @@
     password: "{{ fyreapikey }}"
     force_basic_auth: True
     validate_certs: False
+    return_content: True
   register: clusterStatus
   changed_when: false
   failed_when: false
 
 - name: Create Fyre VM
-  when: clusterStatus.json.status is defined # Status is only defined if cluster does not exist
+  when: clusterStatus.content is not search('deployed')
```

Comment:
what if the cluster exists but is still building?  do you want to still send another request to create a stack with that name?

Reply:
I also hedged the logic by |lower the rerturned content if fyre decides to UPPER CASE anything..